ansible/playbooks/roles/compatibility_layer/tasks/add_overlay.yml

```diff
@@ -58,6 +58,47 @@
         selectattr('eclass-overrides', 'equalto', True) | map(attribute='name') | join(' ')
       }}
 
+- name: Use git instead of rsync for the Gentoo overlay
+  copy:
+    dest: "{{ gentoo_prefix_path }}/etc/portage/repos.conf/gentoo.conf"
+    mode: "0644"
+    content: |
+      [DEFAULT]
+      main-repo = gentoo
+      sync-git-pull-extra-opts = --quiet
+
+      [gentoo]
+      priority  = 1
+      location  = {{ gentoo_prefix_path }}/var/db/repos/gentoo
+      sync-uri  = https://github.com/gentoo/gentoo.git
+      sync-type = git
+      auto-sync = Yes
+      clone-depth = 1
+
+- name: Create repo.postsync.d directory
+  file:
+    path: "{{ gentoo_prefix_path }}/etc/portage/repo.postsync.d"
+    state: directory
+    mode: '0755'
+
+- name: Add post-sync hooks for Gentoo overlay
+  copy:
+    src: "{{ item }}"
+    dest: "{{ gentoo_prefix_path }}/etc/portage/repo.postsync.d/{{ item }}"
+    mode: 0755
+  with_items:
+    - sync_gentoo_cache
+    - sync_gentoo_dtd
+    - sync_gentoo_glsa
+    - sync_gentoo_news
+    - sync_overlay_cache
+    #- sync_gentoo_projects_xml
```

Comment:
@bedroge Why is this commented out? Perhaps clarify in the comment?

Reply:
That file doesn't seem to have execute permission when downloaded from https://github.com/hasufell/portage-gentoo-git-config, which means it won't be run. It downloads this file: https://api.gentoo.org/metastructure/projects.xml, but I'm not sure where this is used. @amadio: would you recommend enabling or disabling this script?

Reply:
This is not essential information for you. It's just a file that contains information about which devs are part of which projects in Gentoo. https://devmanual.gentoo.org/general-concepts/projects/index.html I think some tools use this information (like CI, etc), but for you it's probably fine to not have it.

Reply:
@bedroge Then I would just remove this entry, or at the very least clarify with a comment?

Reply:
I've removed the line. Also just noticed that we still had an old version of the script in the templates dir, removed those.